tests/Build/Changelog/ChangelogBuilderTest.php

```diff
@@ -0,0 +1,141 @@
+<?php
+namespace Aws\Test\Build\Changelog;
+
+use Aws\Build\Changelog\ChangelogBuilder;
+
+class ChangelogBuilderTest extends \PHPUnit_Framework_TestCase
+{
+	/**
+	 * @covers ChangelogBuilder
+	 *
+	 */
+
+	private $RESOURCE_DIR= "tests/Build/Changelog/resources";
+
+	private function getChangelogBuilder()
+	{
+		return new ChangelogBuilder();
+	}
+
```

Comment:
Testing expected exceptions, you could follow: https://phpunit.de/manual/current/en/appendixes.annotations.html#appendixes.annotations.expectedException

Some example in our repo: https://github.com/aws/aws-sdk-php/blob/master/tests/JsonCompilerTest.php#L25